benchmark/buffers/buffer-isutf8.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const buffer = require('node:buffer');
+const assert = require('node:assert');
+
+const bench = common.createBenchmark(main, {
+  n: [2e7],
+  length: ['short', 'long'],
+  input: ['regular string', 'ğ'],
```

Comment:
It might make sense to use something like this https://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt

Reply:
that was the idea of the second input `ğ`. I took it from the tests. But, I can change it.

Reply:
Yes, but with `ğ` we only test a string made of 2 bytes characters.

Reply:
Maybe ` ∀x∈ℝ: ⌈x⌉ = −⌊−x⌋` should make it better?

Reply:
yes, I think so. Using a string made of mixed 1, 2, 3, and 4 bytes would make the benchmark unrealistic but more accurate. See also https://github.com/simdutf/simdutf/pull/81.

Reply:
ps: Any Turkish song lyrics would fit into the `utf8` string category.